content/blog/overview-package-management-python.md

```diff
@@ -0,0 +1,363 @@
++++
+title = "An overview of package management in Python in 2018"
+description = "There are tons of things happening in package management for Python in 2018!"
+date = 2018-09-10
++++
+
+First of all, this article will only talk about package management in the
+context of backend applications. I do not maintain any Python library so I am
+not able to give insights on that side of the package management story.
+
+I will begin by quickly explaining the most common way of setting up Python projects and handling dependencies
+to make sure everyone is on the same page.
+
+## A basic setup
+
+Pretty much all the Django/Flask projects I have seen use the same structure: a `requirements`
+folder containing several `.txt` files listing the dependencies and their versions.
+
+The folder typically looks like the following:
+
+```bash
+.
+├── base.txt
+├── local.txt
+└── prod.txt
```

Comment:
I'm still using `requirements.txt` + `requirements-dev.txt`. Dev installs both, staging & prod only the first one.

Reply:
I just copied the structure from the most used boilerplate: https://github.com/pydanny/cookiecutter-django/tree/master/%7B%7Bcookiecutter.project_slug%7D%7D/requirements